app/assets/stylesheets/custom.css

```diff
@@ -41,4 +41,17 @@ a.black{
 .btn-default{
 	background-color: black;
 	color :white; 
+}
+
+.Pending{
```

Comment:
(Style comment) CSS classes start with lower case. And put a space before the ´{´